lib/publiccloud/provider.pm

```diff
@@ -343,6 +343,14 @@ C<instance_type> defines the flavor of the instance. If not specified, it will l
 =cut
 
 sub create_instance {
+    my $max = get_var('PUBLIC_CLOUD_CREATE_INSTANCE_MAX_CALLS', 1);
```

Comment:
```suggestion
    my $max = get_var('PUBLIC_CLOUD_MAX_INSTANCES', 1);
```
Let's simplify the variable name.

Reply:
Fixed in https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/20721/commits/249338d8c7e973bb2b449d577545ea357c585db5